applications/luci-app-email/htdocs/luci-static/resources/view/email/emailrelay.js

```diff
@@ -0,0 +1,306 @@
+				'Allow connections from the public internet. ' +
+				'<b>You may receive spam so be careful</b>.<br/>' +
+				'Enable ports in firewall.<br/>' +
+				'See <a href="%s#__remote_clients" target="_blank" rel="noreferrer">--remote-clients</a>'
+			).format(docsRef)
+		);
+		o.modalonly = true;
+		o.default = '0';
+
+		o = s.taboption('smtp_server', form.Flag, 'anonymous', _('Anonymous'),
+			_(
+				'Reduce the amount of information leaked to remote clients.<br/>' +
+				'See <a href="%s#__anonymous" target="_blank" rel="noreferrer">--anonymous</a>'
+			).format(docsRef)
+		);
+		o.modalonly = true;
+		o.default = '0';
+
+		o = s.taboption('smtp_server', form.Value, 'domain', _('Domain'),
+			_(
+				'Specifies the server\'s domain name that is used in SMTP EHLO.<br/>' +
+				'By default, the local hostname is used.<br/>' +
+				'See <a href="%s#__domain" target="_blank" rel="noreferrer">--domain</a>'
+			).format(docsRef)
+		);
+		o.datatype = 'hostname';
+		o.optional = true;
+		o.rmempty = false;
+		o.depends('anonymous', '0');
+
+		o = s.taboption('smtp_server', form.Value, 'server_auth', _('Auth file'),
+			_(
+				'Server/proxy authorization file.<br/>' +
+				'See <a href="%s#reference_md_Authentication" target="_blank" rel="noreferrer">Authentication</a>'
+			).format(docsRef)
+		);
+		o.datatype = 'file';
+		o.default = '/etc/emailrelay.auth';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+
+		o = s.taboption('smtp_server', form.Flag, 'server_tls', _('Enable TLS for server'),
+			_(
+				'Use TLS encryption for SMTP and POP connections.<br/>' +
+				'Use <a href="%s" target="_blank" rel="noreferrer">acme.sh to issue a TLS cert</a>.<br/>' +
+				'See <a href="%s#reference_md_TLS_encryption" target="_blank" rel="noreferrer">TLS encryption</a>'
+			).format('https://openwrt.org/docs/guide-user/services/tls/acmesh', docsRef)
+		);
+		o.default = '0';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+
+		o = s.taboption('smtp_server', form.Value, 'server_tls_key', _('TLS private key'),
+			_('Path to TLS private key. ') + _('E.g. <code>/etc/ssl/acme/example.com.key</code>')
+		);
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = true;
+		o.modalonly = true;
+		o.depends('server_tls', '1');
+
+		o = s.taboption('smtp_server', form.Value, 'server_tls_certificate', _('TLS certificate'),
+			_('Path to TLS cert. ') + _('E.g. <code>/etc/ssl/acme/example.com.fullchain.crt</code>')
+		);
+		o.datatype = 'file';
+		o.optional = true;
+		o.rmempty = false;
+		o.modalonly = true;
+		o.depends('server_tls', '1');
+
+		o = s.taboption('smtp_server', form.Value, 'server_tls_verify', _('CA certificate'),
+			_(
+				'Verify an SMTP and POP client\'s certificates against any of the trusted ' +
+				'CA certificates in the specified file or directory.<br/>' +
+				'In many use cases this should be your self-signed root certificate.<br/>' +
+				'Use <code>&lt;default&gt;</code> to use the system trusted CAs'
```

Comment:
Can you move all the sentences separated by `</br>` into its own translation?
So we have:
```
_('Verify an SMTP and POP client\'s certificates against any of the trusted CA certificates in the specified file or directory.) + '<br/>' +
_('In many use cases this should be your self-signed root certificate.) + '<br/>' +
_('Use <code>&lt;default&gt;</code> to use the system trusted CAs.')
```
Could you please update this also for the other Sections in this pullrequest.

This facilitates the translation and the translators do not have to deal with the HTML tags.

Reply:
I checked the Weblate interface to see how it feels for translators.
It shows fields separately but also don't show a place in code so if we have three messages then a translator may not have a clue that they are all related and have a same context.
Short messages would be more likely translated. Bigger messages may be better for performance because less keys. Not sure about this.

The html tags looks like not a problem for the weblate.
Anyway I'll apply the requested changes.